### ISSUE TITLE
Messages disappear before slower readers can finish reading them

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#1118]
Some messages stay on the screen for only one second, too short for older users to finish reading them on time. Therefore there should be a seconds setting for increased accessibility.
![Screenshot_20200302-144715](https://user-images.githubusercontent.com/1230959/75652134-44a88a00-5c95-11ea-8fef-c62d7b6adad8.jpg)

**Comment by simonpoole** [comment]
Standard Android toasts only have two values for length the toast is shown, short and long which is slightly less short. See https://developer.android.com/reference/android/widget/Toast#setDuration(int)

With other words changing the behaviour would require using or building a different mechanism.

**Comment by jidanni** [comment]
Indeed, [tons](https://www.google.com/search?q=android+toast+accessibility) of problems.

**Comment by simonpoole** [comment]
Nothing actionable left here.
